susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -181,6 +181,16 @@ salt-minion-package:
     - makedirs: True
     - require:
       - file: {{ salt_config_dir }}/minion.d/susemanager.conf
+
+{{ salt_config_dir }}/minion.d/beacons.conf:
```

Comment:
I wonder if we should use a custom conf file here, instead of `beacons.conf`. Like:

```suggestion
{{ salt_config_dir }}/minion.d/susemanager_beacons.conf:
```